docs/vdc/networking-virtual-datacenter.md

```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
+[**UDRs**][UDR]. With UDRs, customers can deploy firewalls, IDSs or IPSs, and other virtual appliances. They can route network traffic through these security appliances for security boundary policy enforcement, auditing, and inspection. You can create UDRs in both the hub and the spokes. They guarantee that traffic transits through the specific custom VMs, network virtual appliances, and load balancers used by the VDC. To make sure that traffic generated from VMs resident in the spoke transits to the correct virtual appliances, set a UDR in the subnets of the spoke. Set the front-end IP address of the internal load balancer as the next hop. The internal load balancer distributes the internal traffic to the virtual appliances, or load balancer back-end pool.
 
-[**Azure Firewall**][AzFW] is a managed, cloud-based network security service that protects your Azure Virtual Network resources. It is a fully stateful firewall as a service with built-in high availability and unrestricted cloud scalability. You can centrally create, enforce, and log application and network connectivity policies across subscriptions and virtual networks. Azure Firewall uses a static public IP address for your virtual network resources allowing outside firewalls to identify traffic originating from your virtual network. The service is fully integrated with Azure Monitor for logging and analytics.
+[**Azure Firewall**][AzFW] is a managed, cloud-based network security service that protects your Azure Virtual Network resources. It's a fully stateful firewall as a service with built-in high availability and unrestricted cloud scalability. You can centrally create, enforce, and log application and network connectivity policies across subscriptions and virtual networks. Azure Firewall uses a static public IP address for your virtual network resources. It allows outside firewalls to identify traffic that originates from your virtual network. The service is fully integrated with Azure Monitor for logging and analytics.
 
-[**Network Virtual Appliances**][NVA]
-In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall or a farm of firewalls and/or Web Application Firewalls (WAFs).
+[**Network virtual appliances**][NVA]. In the hub, the perimeter network with access to the internet is normally managed through an Azure Firewall instance or a farm of firewalls or web application firewall (WAF).
 
-Different LOBs commonly use many web applications, and these applications tend to suffer from various vulnerabilities and 
```

Comment:
"However, a single VDC implementation is required to be hosted within a single region because the subscription requirements do not allow spanning of regions. This makes a single VDC implementation vulnerable to regional outages."

Reply:
Replaced the deleted paragraph with this new paragraph.